docs/gettingstarted/modfiles.md

```diff
@@ -2,30 +2,40 @@
+| `loader_version_range`    | The version range of the mod loader this mod can use, as a [Maven Version Range][mvr]. Note that the loader versioning is decoupled from NeoForge versioning.                                                                                         | `loader_version_range=[1,)`               |
+| `mod_id`                  | See [The Mod ID][modid].                                                                                                                                                                                                                              | `mod_id=examplemod`                       |
+| `mod_name`                | The human-readable display name of your mod. By default, this can only be seen in the mod list, however, mods such as [JEI][jei] prominently display mod names in item tooltips as well.                                                              | `mod_name=Example Mod`                    |
+| `mod_license`             | The license your mod is provided under. It is suggested that this is set to the [SPDX identifier][spdx] you are using and/or a link to the license. You can visit https://choosealicense.com/ to help pick the license you want to use.               | `mod_license=MIT`                         |
+| `mod_version`             | The version of your mod, shown in the mod list. See [the page on Versioning][versioning] for more information.                                                                                                                                        | `mod_version=1.0`                         |
+| `mod_group_id`            | See [The Group ID][group].                                                                                                                                                                                                                            | `mod_group_id=com.example.examplemod`     |
+| `mod_authors`             | The authors of the mod, shown in the mod list.                                                                                                                                                                                                        | `mod_authors=ExampleModder`               |
+| `mod_description`         | The description of the mod, as a multiline string, shown in the mod list. Newline characters (`\n`) can be used and will be replaced properly.                                                                                                        | `mod_authors=Example mod description.`    |
+| `pack_format_number`      | The version number of your mod's data and resource pack. Mojang bumps this without something that could be considered a consistent scheme, so it's best to just look up what the current number is. As of Minecraft 1.20.2, the pack version is `18`. | `pack_version_number=18`                  |
```

Comment:
Honestly, we should remove this as the pack format can be datagenned and that was the recommended way until NeoForge removed the requirement.

Actually, on another look, this has been removed from the MDK, so this can be removed here.